app/models/vm_or_template.rb

```diff
@@ -237,6 +237,8 @@ def from_infra_manager?
     'vmsafe.timeoutMS'    => [:vmsafe_timeout_ms,    :integer],
     'entitled_processors' => [:entitled_processors,  :float],
     'processor_type'      => [:processor_share_type, :string],
+    'pin_policy'          => [:pin_policy,           :string],
+    'software_licenses'   => [:software_licenses,    :string],
```

Comment:
@agrare This seems weird to be under "REQUIRED_ADVANCED_SETTINGS".  In fact all of them are weird.  What are your thoughts here?

Reply:
I wonder if this should be something like a `expose_advanced_setting` class method, and then somehow we can pluggably include there from the child classes.

Reply:
Yes I agree that this isn't pluggable at all.  Think there are two issues, 1. required_advanced_settings is weird because it doesn't tell you required for what? 2. this isn't pluggable at all.

We might run into a similar issue with VENDOR_TYPES where you can't use leaf_subclasses because it introduces an infinite loop during constant loading.